src/script/guards/Message.ts

```diff
@@ -17,12 +17,19 @@
  *
  */
 
+import {Draft} from 'Util/DraftStateUtil';
+
+import {ContentMessage} from '../entity/message/ContentMessage';
+import {Message} from '../entity/message/Message';
 import {MemberMessage} from '../entity/message/MemberMessage';
+import {SystemMessage} from '../entity/message/SystemMessage';
 import {SuperType} from '../message/SuperType';
-import {Draft} from 'Util/DraftStateUtil';
+
+export const isContentMessage = (
+  message: Message | ContentMessage | MemberMessage | SystemMessage,
+): message is ContentMessage => message.super_type === SuperType.CONTENT;
+
+export const isDraftMessageWithReplyId = (message: Draft): message is Draft => 'messageId' in message.reply;
```

Comment:
explicit return type doesn't make sense here imo: you already require to pass `Draft` type, so it's always `Draft`, you are just checking if it has messageId field. I would just remove the return type here 🤔